platform_versioned_docs/version-24.3/getting-started/studios.mdx

```diff
@@ -0,0 +1,452 @@
+
+- **Region**: To reduce costs, your compute environment should be in the same region as your data. To browse the 1000 Genomes public data optimally, select **us-east-1**.
+- **Provisioning model**: Use **On-demand** EC2 instances. 
+- Studios does not support AWS Fargate. Do not enable **Use Fargate for head job**. 
+- At least 2 available CPUs.
+
+#### Add data using Data Explorer
+
+Add the 1000 Genomes S3 bucket to your workspace using Data Explorer:
+
+1. From the **Data Explorer** tab, select **Add cloud bucket**. 
+1. Specify the bucket details:
+    - **Provider**: AWS
+    - **Bucket path**: `s3://1000genomes`
+    - A unique **Name** for the bucket, such as `1000G`.
+    - **Credentials**: Select **Public** from the dropdown menu.
+    - An optional bucket **Description**.
+1. Select **Add**.
+
+:::info 
+To use your own data for interactive analysis, see [Add a cloud bucket](./quickstart-demo/add-data.mdx#add-a-cloud-bucket) for instructions to add your own public or private cloud bucket. 
+:::
+
+#### Create an Xpra studio 
+
+From the **Data Studios** tab, select **Add a data studio** and complete the following:
+- In the **Compute & Data** tab:
+    - Select your AWS Batch compute environment. 
+      :::note
+      Studios compete for computing resources when sharing compute environments. Ensure your compute environment has sufficient resources to run both your pipelines and data studio sessions. The default CPU and memory allocation for a studio is 2 CPUs and 8192 MB RAM. 
+      :::
+    - Optional: Enter CPU and memory allocations.  
+    - Mount the 1000 Genomes S3 bucket you added previously using Data Explorer. 
+- In the **General config** tab:
+    - Select the latest **Xpra** container image template from the list.
+    - Optional: Enter a unique name and description for the studio. 
+    - Check **Install Conda packages** and paste the following into the YML field:
+        ```yaml 
+        channels:
+          - conda-forge
+          - bioconda
+        dependencies:
+          - igv
+          - samtools
+        ```
+- Select **Add** or choose to **Add and start** the studio immediately.
+- If you chose to **Add** the studio in the preceding step, select **Connect** in the options menu to open the studio in a new browser tab. 
+
+#### View variants in IGV desktop
+
+1. In the Xpra terminal, run `igv` to open IGV desktop.
+1. In IGV, change the genome version to hg19.
+1. Select **File**, then **Load from file**, then navigate to `/workspace/data/xpra-1000Genomes/phase3/data/HG00096/high_coverage_alignment` and select the `.bai` file, as shown below:
+    ![Load BAM file in IGV desktop](./_images/xpra-data-studios-IGV-load-bam.png)
+1. Search for PCSK9 and zoom into one of the exons of the gene. A coverage graph and reads should be shown, as below:
+    ![BAM file view](./_images/xpra-data-studios-IGV-view-bam.png)
+
+### VS Code: Create an interactive Nextflow developer environment
```

Comment:
IIRC VS Code has about a bazillion ways to import user preferences / configurations. Does the tutorial need to consider this angle (e.g. if I have a very customized local VS Code for executing my workflows and I now want to port this to a DS session, I assume I'd want my customizations to come with me?